zinc/src/sbt-test/source-dependencies/nested-type-params/test

```diff
@@ -0,0 +1,8 @@
+> compile
+
+$ copy-file changes/Providers.scala Providers.scala
+$ copy-file changes/Bar.scala Bar.scala
+
+-> compile # This should compile
```

Comment:
@jvican this compile should succeed (clean build 2 lines below is green). I prefer this style of pointing out issues since if I leave it as a last command and change `test` to `pending` *any* failure  will make this test green. There is an issue for this and if someone fix the problem this test will fail and comment in this line will tell that person that this is expected state.

Previously tests was green since there was 3 files and changing 2 of them was over 50% and problematic compilation was in fact clean-build.

Reply:
The issue with this approach is that we cannot know just by grepping through the directory which tests are failing and which are not, though.

Reply:
TBH I can chabge is to pending - its up to you.

Reply:
I can see both sides. Let's establish a keyword to use in a comment, which we can grep for.

Reply:
Perhaps `PENDING`

Reply:
Your solution is good. Let me propose something similar that I think it's better: let's change `pending` to mean "there's one or more statements in this test case that fail but should succeed". How do we mark the statements that are "PENDING"? Well, I have two suggestions:
  
This requires a change in the scripted implementation, but shouldn't take long. Could you do it @romanowski ? Essentially

Reply:
Maybe we can keep it even simpler:
```
PENDING > compile
PENDING $ copy-file
PENDING -> clean
# etc
````

 I can take a look tomorrow if we agree on notation.

Reply:
I don't mind syntax, your take @dwijnand?

Reply:
I like Krzysztof's `PENDING >`. Easier to intuit, easier to grep for.

Reply:
Looking forward to this change and merging the pending test. I'll take a stab at it this w-e.

Reply:
Making this change is too much of a hassle. Let's merge this and I'll follow up with a fix in another PR.